tests/Hangfire.PostgreSql.Tests/PostgreSqlJobQueueFacts.cs

```diff
@@ -468,11 +468,16 @@ public void Queues_Can_Dequeue_On_Signal()
     [CleanDatabase]
     public void Queues_Can_Dequeue_On_Notification()
     {
-      var timeout = TimeSpan.FromSeconds(30);
-
       UseConnection((connection, storage) => {
 
-        storage.Options.QueuePollInterval = TimeSpan.FromMinutes(2);
+        var timeout = TimeSpan.FromSeconds(30);
+
+        // Only for Postgres 11+ should we have a polling time greater than the timeout.
+        if (((Npgsql.NpgsqlConnection)connection).PostgreSqlVersion.Major > 10)
```

Comment:
I'd like to change the checks to `>= 11` both here and in `SupportsNotifications`. A bit clearer in my opinion.